<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget_Main</class>
 <widget class="QWidget" name="Widget_Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_Chat" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>161</width>
     <height>471</height>
    </rect>
   </property>
   <widget class="QWidget" name="widget_ChatList" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>120</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_ChatHead" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="lineEdit_search">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Add">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_Tools" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>31</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_Message" native="true">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>521</width>
     <height>531</height>
    </rect>
   </property>
   <widget class="QWidget" name="widget_MessageShow" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>371</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_MessageHead" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_MessageSend" native="true">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>350</y>
      <width>501</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
